src/HW10.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Arrays;
+import java.util.Random;
+
+public class HW10 {
+    public static void main(String[] args) {
+        int[] lazyArray = new int[7];
+        int[] lazyArray2 = new int[7];
+        Random random = new Random();
+        int count = 0;
+        for (int i = 0; i < lazyArray.length; i++) {
+            lazyArray[i] = random.nextInt(9);
```

Comment:
Max number from the random is 8, when should be 9

Reply:
Hello, I see only number 9.
Where I wrote number 8?